index.html

```diff
@@ -1226,12 +1215,18 @@ <h2 class="intro-text text-center">Top 10 par année</h2>
     <script type="text/javascript" src="data/medal-map-data.js"></script>
     <!-- Scripting goes here -->
 
-
-
     <script src="https://code.highcharts.com/maps/highmaps.js"></script>
     <script src="https://code.highcharts.com/maps/modules/exporting.js"></script>
     <script src="https://code.highcharts.com/mapdata/countries/fr/custom/fr-all-mainland.js"></script>
 
+
+    <script src="js/skel.min.js"></script>
+    <script src="js/skel-layers.min.js"></script>
+    <script src="js/init.js"></script>
+
+    <!-- Chargements des scripts de Claudia pour les cartes -->
+    <script src="js/year_carte.js"></script>
```

Comment:
It's better to put the scripts at the bottom of the page: when the browser read them, the page is already rendered, and you can use safely things like `$('#my-element')` to retrieve and modify them. (Otherwise, they might not be present yet!)